specs/opcode/F1CALL_F2CALLCODE_F4DELEGATECALL_FASTATICCALL.md

```diff
@@ -4,7 +4,12 @@
 
 ### EVM behavior
 
-The `CALL` opcode transfers specified amount of ether to callee and creates a new call context and switch to it. This is done by popping serveral words from stack:
+- for opcode `CALL`, transfer specified amount of ether to callee and creates a new call context and switch to it
```

Comment:
This seems to be outdated, changes in this file could be reverted.

Reply:
I just arranged the layout in order to understand easily.
Do you mean the spec in this file doesn't reflect current spec of zkevm-circuits?

Reply:
Ah I see, sorry for misunderstanding.